src/R7InsightNLog/InsightTarget.cs

```diff
@@ -13,115 +13,141 @@
 namespace NLog.Targets
 {
     [Target("Insight")]
-    public sealed class InsightTarget : TargetWithLayout
+    public sealed class InsightTarget : TargetWithLayout, IAsyncLoggerConfig
     {
-        private AsyncLogger insightAsync;
+        class NLogAsyncLogger : AsyncLogger
+        {
+            protected override void WriteDebugMessages(string message, Exception ex)
+            {
+                base.WriteDebugMessages(message, ex);
+                InternalLogger.Warn(string.Concat(message, " Exception: ", ex.ToString()));
+            }
+
+            public override bool LoadCredentials()
+            {
+                bool success = base.LoadCredentials();
+                if (!success)
+                {
+                    InternalLogger.Warn("Failed to load credentials for LogEntries (GET), please check LOGENTRIES_TOKEN configuration");
```

Comment:
This error message is not correct in every case. For example: I left the region option empty, but the token was there and still got the same error message. I wasn't able to see the error message related to missing region too. Maybe we can set this a more general error. What do you think? @snakefoot

Reply:
@stopal-r7 Since you have not proposed any alternative. Then I suggest that I remove the following `for LogEntries (GET), please check LOGENTRIES_TOKEN configuration`. Then the user can do their own random guessing.

Reply:
Sounds good!

Reply:
Fixed